### ISSUE TITLE
how to add video from direct video source url

### ISSUE THREAD

**Issue by balrng** [avscms/avscms#38]
Hello ,

Is there a option to add video by providing video url like=>
xxxxx.com/videos/test1.flv

Many free scripts provide such options.

**Comment by iwebhostu** [comment]
embedded plugin

**Comment by balrng** [comment]
No, i don't want to get and paste embed code. Some free scripts have options to get video file url (also captured video lenght,creating auto thumbnails from the video ) like domain.com/video.flv then convert it via FFMpeg/Mplayer and also creating thumbnails from the video source directly. Do you provide such extension plugin?

**Comment by iwebhostu** [comment]
I think you have 2 options here
1. https://www.avscms.com/plugins/csv-importer = here you can create a CSV file with title,description,thumb.jpg and video.mp4 nad upload them.
2. OR https://www.avscms.com/plugins/video-grabber = Here a website was SlutLoad (I don't it's exists or not now) but I upload videos here and import them on my website. I just had a shared hosting account then so I don't need ffmpeg or other tools.

It's on you!
